tests/testthat/apps/pws_download_ggplot2/tests/testthat/test-shinytest2.R

```diff
@@ -0,0 +1,87 @@
+library(shinytest2)
+
+# download as .png
+test_that("{shinytest2} recording: pws_dwnl", {
+  skip_on_cran()
+  skip_on_ci()
+
+  app <- AppDriver$new(name = "pws_dwnl", height = 937, width = 1619)
+
+  # download plot as initialized
+  # saving images or svg (below) might be brittle
+  # i wonder if the 'compare' argument of AppDriver$expect_download can
+  #  help here
+  # another option might be to use shiny::exportTestValues() within the module
+  #  server
+  # then we could check information about the plot (class, input data, ...)
+  # in that case, maybe an expect_error(app$get_download('..'), NA) could
+  #  suffice for download functionality
+  app$click("plot_with_settings-downbutton-downl")
+  app$set_inputs(`plot_with_settings-downbutton-downl_state` = TRUE)
+  app$set_inputs(`plot_with_settings-downbutton-file_name` = "plot_1")
+  app$expect_download("plot_with_settings-downbutton-data_download")
+
+  plot_name1 <- app$get_value(input = "plot_with_settings-downbutton-file_name")
+  testthat::expect_equal(plot_name1, "plot_1")
+
+  app$set_inputs(`plot_with_settings-downbutton-downl_state` = FALSE)
+
+  # why the change in window size?
```

Comment:
@asbates when one click the download button, in chrome window resizes itself a little bit. So it is a recording of the thing that actually happened.